src/components/books/books.js

```diff
@@ -0,0 +1,18 @@
+import { useSelector } from 'react-redux';
+import Book from './book';
+import BooksForm from './booksForm';
+
+const Books = () => {
+  const books = useSelector((state) => state.books);
+
+  const renderedBooks = books.map((book) => <Book key={book.id} book={book} />);
+
+  return (
+    <div className="books">
+      <ul className="books-list">{renderedBooks}</ul>
```

Comment:
- [ ] I see you have the code to show the books, but when I start the application, I wasn't able to see the list, please add the books list on the start project, this could help to see how is it and be more ready for next steps. 🐶

Reply:
* Done 👍